src/components/link/index.tsx

```diff
@@ -50,8 +52,8 @@ const Link = forwardRef<HTMLAnchorElement, Props>(
 
     return (
       <Component
-        target={isExternal ? '_blank' : undefined}
-        rel={rel || isExternal ? 'noopener noreferrer' : undefined}
+        target={target || (isExternal ? '_blank' : undefined)}
+        rel={rel || (isExternal ? 'noopener noreferrer' : undefined)}
```

Comment:
This means that if I set both `isExternal` and `target` or `rel` I would override the values. It potentially allows me to skip `noopener` and `noreferrer` or open the external link in the same browser tab. Is this intended? Maybe it makes sense to re-think the `isExtetnal` and extend or remove it altogether for simplicity?

Reply:
@Averethel  If I extend it or remove it then we will need to consider to sync the Button component as well because there we are using the isExternal and rel patterns as well. 
I will keep it simple and do the same as we are doing for the button component (the conditions). In case that we want to make the Link component more restricted, I can create a follow up ticket for readjusting the Link and Button components.
What do you think?